js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+let navAnchors = document.querySelectorAll('a');
+let mainHeader = document.querySelector('h1');
+let headerButton = document.querySelector('button');
+let headerImg = document.getElementById('cta-img');
+let H4 = document.querySelectorAll("h4");
+let paragraphs = document.querySelectorAll("p")
+let middleImg = document.getElementById("middle-img");
+
+
+let navList = document.querySelectorAll('nav a')
+for (let i = 0; i < navList.length; i++) {
+  navList[i].textContent = siteContent['nav'][`nav-item-${i + 1}`];
+  navList[i].style.color = 'green';
+};
+
+// new content added using prepend and append
+const newNavList = document.querySelector('nav');
+const navHome = document.createElement('a');
+const navBlog = document.createElement('a');
+navHome.href = '#';
+navHome.textContent = 'Home';
+navHome.style.color = 'green';
+navBlog.href = '#';
+navBlog.textContent = 'Blog';
+navBlog.style.color = 'green';
+
+newNavList.appendChild(navBlog);
+newNavList.prepend(navHome);
+
+//Header
+mainHeader.textContent = siteContent["cta"]["h1"];
+
+headerImg.setAttribute('src', siteContent["cta"]["img-src"]);
+headerButton.textContent = siteContent["cta"]["button"];
+
+H4[0].textContent = siteContent["main-content"]["features-h4"];
+H4[1].textContent = siteContent["main-content"]["about-h4"];
+H4[2].textContent = siteContent["main-content"]["services-h4"];
+H4[3].textContent = siteContent["main-content"]["product-h4"];
+H4[4].textContent = siteContent["main-content"]["vision-h4"];
+
+
+//Main Content
+paragraphs[0].textContent = siteContent["main-content"]["features-content"];
+paragraphs[1].textContent = siteContent["main-content"]["about-content"];
+paragraphs[2].textContent = siteContent["main-content"]["services-content"];
+paragraphs[3].textContent = siteContent["main-content"]["product-content"];
+paragraphs[4].textContent = siteContent["main-content"]["vision-content"];
+middleImg.setAttribute('src', siteContent["main-content"]["middle-img-src"]);
```

Comment:
We can use a very similar pattern here to loop through just the `content` keys. 🙂